### PR TITLE
JAX-WS: Add PartInfoNamespaceCorrectionTest for fixes to CXF's SoapBindingFactory

### PR DESCRIPTION
**Pull request by neuwerk**
This PR adds the following test `PartInfoNamespaceCorrectionTest` to the `com.ibm.ws.jaxws.2.2.webcontainer_fat` bucket.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/servlet/PartInfoNamespaceCorrectionTestServlet.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+            Holder<PersonType> hpt = new Holder();
+            PersonType pt = new PersonType();
+            pt.setName("John Doe");
+            pt.setEmail("john.doe@dummy.email.com");
+            hpt.value = pt;
+            proxy.nameSpaceCheck(nameSpaceCheckMessage, hpt);
+            fail("CustomFaultDeatil was not the caught, and no exception was returned by the provider");
+        } catch (CustomFaultDetail e) {
+            // TODO Auto-generated catch block
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/servlet/PartInfoNamespaceCorrectionTestServlet.java]: auto generated comment

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/CustomFaultDetail.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/CustomFaultDetail.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/PersonType.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/PersonType.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/ObjectFactory.java [hunk 4]
```diff
@@ -0,0 +1,170 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/ObjectFactory.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/CustomHeaderType.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/CustomHeaderType.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/CustomFaultType.java [hunk 6]
```diff
@@ -0,0 +1,90 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/importedserviceschema/CustomFaultType.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckPortType.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckPortType.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckService.java [hunk 8]
```diff
@@ -0,0 +1,89 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckService.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckService_NameSpaceCheckServicePortHttpImpl.java [hunk 9]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainservice/NameSpaceCheckService_NameSpaceCheckServicePortHttpImpl.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/DummyObjectTYPE.java [hunk 10]
```diff
@@ -0,0 +1,91 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/DummyObjectTYPE.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/NameSpaceCheckMessage.java [hunk 11]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/NameSpaceCheckMessage.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/NameSpaceCheckResponseMessage.java [hunk 12]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/NameSpaceCheckResponseMessage.java]: copyright header

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/ObjectFactory.java [hunk 13]
```diff
@@ -0,0 +1,144 @@
+
```
Comment (WhiteCat22) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/test-applications/namespaceCheck/src/com/ibm/ws/jaxws/namespacecheck/wsdl/mainserviceschema/ObjectFactory.java]: copyright header

**Review by WhiteCat22 — CHANGES_REQUESTED** [review]
I did a sweep of the namespace URLs and I didn't notice anything troublesome

**Review by WhiteCat22 — APPROVED** [review]

**Comment by neuwerk** [comment]
#libby

**Pull request merged**